config/complaint-form-email-converter/templates/utils.js

```diff
@@ -13,3 +15,12 @@ export function humanReadableSize(size) {
   const i = parseInt(Math.floor(Math.log(bytes) / Math.log(1024)));
   return Math.round(bytes / Math.pow(1024, i), 2) + ' ' + sizes[i];
 }
+
+/** @arg {Date} date */
+export function formatDate(date) {
+  return formatInTimeZone(
+    date,
+    'Europe/Brussels',
+    'dd/MM/yyyy HH:mm'
+  );
+}
```

Comment:
We do not have a formatter to enforce these rules, so I am not sure if this comment is actionable. But, I noticed the other two `.js` files in this PR have a new-line at the end, whereas this one does not. It is difficult to decide whether to add it here or not with the absence of formatting rules, but I figured I would mention it just in case.

Reply:
Yea, developing this was a pain without a linting / formatting / .editorconfig setup.

I don't think the app-* repos are intended for development, but simply a grouping of configs. I'm not sure why this service works this way either.. It depends on us adding the needed js files in the config folder from the app- repo. It feels very brittle and unnecessary, especially since it's custom made for this project.

Personally, I think we should just "hardcode" this code into the service itself.

Anyways, good catch, I did add the newline now 👍.